guidelines.md

```diff
@@ -76,7 +76,80 @@ The enterprise is based in UK, it accepts `GBP`, `EUR` and `USD` currencies \(an
 
 * **PlatformAddress** - `https://www.mews.li`
 * **ClientToken** - Will be provided to you by our integration team. For further information, please contact [integrations@mewssystems.com](mailto://integrations@mewssystems.com).
-* **AccessToken** - Depends on the enterprise, should be provided to you by the enterprise admin.
+* **AccessToken** - Will be provided to you via [email and your provided webhook](guidelines.md#AccessToken-delivery).
+
+### Integration event webhook
+
+The integrator may provide a webhook address to [integrations@mewssystems.com](mailto://integrations@mewssystems.com) where Mews will deliver the `AccessToken` upon integration creation and other events related to the integration.
```

Comment:
An access token is not delivered every time. Where Mews will push the data related to integration operation? Or something like that?